Message-Id: <0cdc51110bdf249013fe7a6fbe59643ea8fb406b.1336068799.git.agx@sigxcpu.org>
In-Reply-To: <49b139fec658b533e7a58a5942dae67f3cdc365b.1336068799.git.agx@sigxcpu.org>
References: <5909887366833c92178e3ed650b87d666f9f50c1.1336068799.git.agx@sigxcpu.org>
	<56db9d711b7615ece3848c57f71f362f0e5f4fea.1336068799.git.agx@sigxcpu.org>
	<727a1c60fdc68e24b2f457f5b7e3611b2aae7a7b.1336068799.git.agx@sigxcpu.org>
	<af426bb880140c2941366a3a84fd11a714b9c778.1336068799.git.agx@sigxcpu.org>
	<64360f6da8f2d82618e0eda5d971ee6a083851ae.1336068799.git.agx@sigxcpu.org>
	<308046ae1d09cc7cce80fdb268cfb7fee35de13d.1336068799.git.agx@sigxcpu.org>
	<f96b7f473f61a8e6d4866a49d9fcc861bed443b5.1336068799.git.agx@sigxcpu.org>
	<7e773c9b5d95a42321be6b5fc7a11cf0fd51a124.1336068799.git.agx@sigxcpu.org>
	<6d5763889d5f114ed6f149ecd47c562f06868519.1336068799.git.agx@sigxcpu.org>
	<e82f2035cd9341f3530199e68c11a27790f041de.1336068799.git.agx@sigxcpu.org>
	<87e0b605aa093d95c507083de601261098cd1aff.1336068799.git.agx@sigxcpu.org>
	<d7d5684690a5110e57b4f5a5ba48b76aff0f183c.1336068799.git.agx@sigxcpu.org>
	<574c21aa539700de7e6214f0123644312d1724b3.1336068799.git.agx@sigxcpu.org>
	<414a2a09a0c8f8b00c5804b2d5afd2ae704fd96f.1336068799.git.agx@sigxcpu.org>
	<bed21802c06c687175cf38a4a62f586a9945d804.1336068799.git.agx@sigxcpu.org>
	<790f6e636b6caf5fa02e645ea0bc473892a0b86e.1336068799.git.agx@sigxcpu.org>
	<6e9717dcb665e73ae0c596d35f77e4c60db42f5d.1336068799.git.agx@sigxcpu.org>
	<aa933d6bee9818778dd49d18cac78f61e237ae18.1336068799.git.agx@sigxcpu.org>
	<54674a84ec4ad50d943b33191aa1ffa48f35e749.1336068799.git.agx@sigxcpu.org>
	<37a22afe7d88ed8395011dc0c01de7bb8671a40d.1336068799.git.agx@sigxcpu.org>
	<601f56221d0e98cb33e7d4cec1a5d37586e4a64a.1336068799.git.agx@sigxcpu.org>
	<66a30ba49e8b45c80dfde1f2921988ce1964b97f.1336068799.git.agx@sigxcpu.org>
	<7157c037befeb30f9ae8b92232ecd4e2639d166a.1336068799.git.agx@sigxcpu.org>
	<a60d095ea97fbf436597b4bb0e19802364e60a3c.1336068799.git.agx@sigxcpu.org>
	<32e4c48a65f59ab8c3868ab05bed1a7b98811908.1336068799.git.agx@sigxcpu.org>
	<a0de06625d26cfcf94ddd1b73a6ff83867433dd3.1336068799.git.agx@sigxcpu.org>
	<49b139fec658b533e7a58a5942dae67f3cdc365b.1336068799.git.agx@sigxcpu.org>
From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 2 Apr 2011 10:37:17 +0200
Subject: Brand zh-TW

---
 zh-TW/locale/zh-TW/calendar/migration.properties  |    2 +-
 zh-TW/locale/zh-TW/lightning/lightning.dtd        |    2 +-
 zh-TW/locale/zh-TW/lightning/lightning.properties |    6 +++---
 3 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/zh-TW/locale/zh-TW/calendar/migration.properties b/zh-TW/locale/zh-TW/calendar/migration.properties
index fd28495..4b1b891 100644
--- a/zh-TW/locale/zh-TW/calendar/migration.properties
+++ b/zh-TW/locale/zh-TW/calendar/migration.properties
@@ -43,4 +43,4 @@ migrationTitle = %1$S: 匯入資料
 migrationDescription=%1$S 可以從許多受歡迎的應用程式匯入行事曆資料。已從您的電腦發現下列應用程式的資料，請選取想要從哪一個應用程式匯入資料。
 finished = 完成
 disableExtTitle = 發現不相容的擴充程式
-disableExtText = 您已安裝了與 Lightning 不相容的舊版 Mozilla Calendar 擴充程式。將會停用該程式並重新啟動 %1$S。
+disableExtText = 您已安裝了與 Iceowl 不相容的舊版 Mozilla Calendar 擴充程式。將會停用該程式並重新啟動 %1$S。
diff --git a/zh-TW/locale/zh-TW/lightning/lightning.dtd b/zh-TW/locale/zh-TW/lightning/lightning.dtd
index 30daaf2..8b06d8f 100644
--- a/zh-TW/locale/zh-TW/lightning/lightning.dtd
+++ b/zh-TW/locale/zh-TW/lightning/lightning.dtd
@@ -46,7 +46,7 @@
   -->
 
 <!-- Tools menu -->
-<!ENTITY lightning.taskLabel "Lightning">
+<!ENTITY lightning.taskLabel "Iceowl">
 
 <!-- New menu popup in File menu -->
 <!ENTITY lightning.menupopup.new.event.label    "事件…">
diff --git a/zh-TW/locale/zh-TW/lightning/lightning.properties b/zh-TW/locale/zh-TW/lightning/lightning.properties
index 845737e..77eceec 100644
--- a/zh-TW/locale/zh-TW/lightning/lightning.properties
+++ b/zh-TW/locale/zh-TW/lightning/lightning.properties
@@ -38,12 +38,12 @@
 # ***** END LICENSE BLOCK *****
 
 # addon description:
-extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.name=Lightning
+extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.name=Iceowl
 extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.description=An integrated calendar for Thunderbird
 extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.creator=Mozilla Calendar Project
 
 # Lightning branding
-brandShortName=Lightning
+brandShortName=Iceowl
 
 # Task mode title
 taskModeApplicationTitle=工作
@@ -72,7 +72,7 @@ imipBarRequestText=此郵件包含事件的邀請。
 imipBarUpdateText=此郵件包含現有事件的更新。
 imipBarAlreadyProcessedText=此郵件包含一已經處理過的事件。
 imipBarReplyText=此郵件包含對邀請的回覆。
-imipBarUnsupportedText=此郵件包含此版 Lightning 無法處理的事件。
+imipBarUnsupportedText=此郵件包含此版 Iceowl 無法處理的事件。
 imipBarProcessingFailed=郵件處理失敗。狀態: %1$S。
 imipAcceptInvitation.label=接受
 imipCancelInvitation.label=刪除
